### DISCUSSION TITLE
Hwp파일을 불러와서 다른이름으로 저장(내보내기)가 가능할까요?

### DISCUSSION THREAD

**Discussion by SID9-HoneyBee** [hancom-io/hwpx-owpml-model#3] [Q&A]
현재 OCX 를 이용해서 동일한 기능을 구현해서 사용하고있습니다 . C# 으로 개발해서 사용중이며
로컬에 저장된 한글파일을 불러와서 이미지나 PDF 로 저장하는 형태로 활용중입니다.

OCX 를 더이상 사용 할 수 없게 된다는 이슈등에 대비하고자 정보를 수집하고 있는 중인데, 
혹시 본 라이브러리로 HWP(Hwpx) 파일을 다른형태(이미지나 PDF) 로 내보내기를 수행할 수 있을까요?

**Comment by hnc-jjeong** [comment]
안녕하세요.

hwpx-owpml-model은 OWPML파일 필터에 대한 오픈소스로 다른 파일 필터는 지원하지 않기 때문에 다른 형태로 저장하는 것은 어렵습니다.
